omero/developers/PythonBlitzGateway.txt

```diff
@@ -83,6 +145,21 @@ have on the client-side will become stale. We will attempt to document
 this a little better in due course, but our general advice is to create,
```

Comment:
found that a bit strange, it sounds like we did not bother explaining
I will only keep ``our general advice is to create, use and close the stateful services in the shortest practicable time``